src/index.js

```diff
@@ -1,11 +1,47 @@
 import React from 'react';
-import ReactDOM from 'react-dom/client';
 import './index.css';
+import * as ReactDOM from 'react-dom/client';
+import {
+  createBrowserRouter,
+  RouterProvider,
+} from 'react-router-dom';
 import App from './App';
+import reportWebVitals from './components/reportWebVitals';
+import Quote from './components/Quote';
+import Calculator from './components/Calculator';
+import Home from './components/Home';
+import PageError from './components/error';
 
-const root = ReactDOM.createRoot(document.getElementById('root'));
-root.render(
+const router = createBrowserRouter([
+  {
+    path: '/',
+    element: <App />,
+    errorElement: <PageError />,
+    children: [
+      {
+        path: '/',
+        element: <Home />,
+      },
+      {
+        path: 'Home/',
```

Comment:
## OPTIONAL 

- [ ] For best practice I think it's good to routes name to be in small letter like `home` not `Home`